.codecov.yml

```diff
@@ -0,0 +1,13 @@
+codecov:
+  require_ci_to_pass: yes
+
+coverage:
+  precision: 2
+  round: down
+  range: "70...100"
```

Comment:
I don't like suggesting 100% coverage as green.  I think 90 would be good here. This also can be adjusted once we see what our current coverage is and what we think it could be.